File: Algorithms/test_sort.py
from sort import bubble_sort, insertion_sort


if __name__ == "__main__":
    array = [9, 4, 3, 0, 1]
    bubble_sort(array, is_descend=True)
    print(array)

    bubble_sort(array, is_descend=False)
    print(array)

File: Algorithms/test_sort.cpp
#include <iostream>

#include "sort/sort.h"

int main()
{
    const uint length = 5;
    int array[length] = {8, 2, 4, 0, 1};
    BubbleSort(array, length, false);
    for (int i = 0; i < length; i++) {
        std::cout << array[i] << std::endl;
    }
}
